templates/common/storybook/.storybook/main.js

```diff
@@ -1,15 +1,15 @@
 module.exports = {
   stories: [
-    "../../apps/**/**/*.story.js",
```

Comment:
@chandrakailash1993  The old scope was correct, We want storybook to look into apps/* and packages/*